src/wasm-binary.h

```diff
@@ -435,7 +436,10 @@ enum Subsection {
   NameElem = 8,
   NameData = 9,
   // see: https://github.com/WebAssembly/gc/issues/193
-  NameField = 10
+  NameField = 10,
+
+  DylinkMemInfo = 1,
```

Comment:
It seems weird to have duplicate values in an enum, especially when they are not intended to be exactly equivalent replacements for each other (i.e. they are used in different contexts). Should we just have a different enum for NameSubsection and DylinkSubsection?

Reply:
Yes that would make sense to me... I'm not sure why it was done this way.